### PR TITLE
made the soft calendar post!

### PR DESCRIPTION
**Pull request by puffballinthedark**
here it is! it's me! with a post! finally!

### CODE REVIEW

**Comment by RamZallan** [comment]
LGTM!

**Comment by csssuf** [comment]
I'm not gonna leave a ton of individual line-by-line comments for it, so I'll just sum it up here:
Capitalize:
* Android
* Google Calendar
* Android Studio

Also in general, I would `s/worked/works/` and bring things into the present tense - they still work the same way, presumably.

File: _posts/2017-11-27-soft-calendar.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+image-sm: https://csh.rit.edu/~wgahnagl/smallerSoftCalendar
+author: Skyler Clark
+author-image: https://avatars1.githubusercontent.com/u/4897492?s=460&v=4
+author-bio: A freshman computer science student!
+author-email: sjc1689@rit.edu
+author-social:
+  github: https://github.com/puffballinthedark
+  linkedin: https://www.linkedin.com/in/skyler-c-58962012b/
+---
+Soft calendar is an android app that helps keep google calendars clean and organized, making an easy to view checklist of events and their due dates! It is integrated with the google calendar, so when you want to commit to an event, just select it from the list and push it to your calendar!
```
Comment (csssuf) [inline _posts/2017-11-27-soft-calendar.md]: Strike "the" in `the google calendar`.

File: _posts/2017-11-27-soft-calendar.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+author-image: https://avatars1.githubusercontent.com/u/4897492?s=460&v=4
+author-bio: A freshman computer science student!
+author-email: sjc1689@rit.edu
+author-social:
+  github: https://github.com/puffballinthedark
+  linkedin: https://www.linkedin.com/in/skyler-c-58962012b/
+---
+Soft calendar is an android app that helps keep google calendars clean and organized, making an easy to view checklist of events and their due dates! It is integrated with the google calendar, so when you want to commit to an event, just select it from the list and push it to your calendar! 
+
+Soft calendar was the first android app I’ve ever built, and with it came a lot of challenges. In all honesty, I would say that the hardest step was configuring my development environment, and learning about how android studio worked! After that, I had to learn about how the google calendar API worked, how authentication worked, how to pass data through  forms, how to save user input, (and fix all of the bugs that naturally crop up along the way when you have no idea what you’re doing!)
```
Comment (calvin-clark) [inline _posts/2017-11-27-soft-calendar.md]: 2 spaces between "through" and "forms, change to one space

**Comment by calvin-clark** [comment]
Overall I think the content is good, but while the tone gives this post a lot of character it's a little excessive with the exclamation points at points, and I agree with the changes James suggested.  Also, the wording of the 2nd-last paragraph might work better if you said "Ironically, ..." instead of "The irony is that" but that's just a minor grammar things.  Other than that it looks good!

**Comment by sgreene570** [comment]
If you could rebase your 2 commits into 1, that'd be lit. Ping me if you need assistance.

**Comment by RamZallan** [comment]
@puffballinthedark  can you make that a more useful commit message? Since it'll be merged into the main blog codebase, it would be better if it described the changes you've made. 

[Here's](https://chris.beams.io/posts/git-commit/) a really helpful post about commit messages!

**Pull request merged**